joomla/joomla-cms
# Firefox subform drag fix

Pull Request for Issue #37974.

### Summary of Changes

Issue looks to be caused by a bug with Firefox

- https://stackoverflow.com/questions/15967786/unable-to-drag-a-button-element-in-firefox
- https://bugzilla.mozilla.org/show_bug.cgi?id=568313

Reworked subform drag button styling, increasing span click area to circumvent browser bug.

### Testing Instructions

Test in multiple browsers **including** Firefox
1. Open the 'System - HTTP Headers' plugin
2. Add at least 2 subform fields to 'Force HTTP Headers'
3. Check drag button functions correctly

### Actual result BEFORE applying this Pull Request

Only part of the subform drag button functioned in Firefox

### Expected result AFTER applying this Pull Request

Drag button is fully clickable in all browsers

### Link to documentations
Please select:
